lib/Layout.js

```diff
@@ -2354,9 +2354,11 @@ class Blob extends Layout {
     if ((offset + span) > b.length) {
       throw new RangeError('encoding overruns Uint8Array');
     }
-    uint8ArrayToBuffer(b).write(src.toString('hex'), offset, span, 'hex');
+    const buffer = uint8ArrayToBuffer(b);
```

Comment:
In the earlier commit you use:

    b = uint8ArrayToBuffer(b);

I'd rather see consistency so please change this to match.  Also `srcBuffer` should just change the variable `src`.

Also: why is this necessary?  The test seems to pass at the commit before this; if this is necessary there should be a new test case that fails without this change to demonstrate what's going on.

Depending on what motivated this change, it might be squashed into the one that changes all the other uses, so there bisecting wouldn't hit a commit that has a bug that resulted from the change in underlying implementation.

Reply:
Motivation of this change is that if the API supports `Uint8Array` params, we need to polyfill the `Buffer.write` method which doesn't exist on `Uint8Array`.

Reply:
Added a new test case that fails without the conversion for `b` and `src`